pkg/driver/clickhouse/cluster_config/ch-cluster-01/config.xml

```diff
@@ -0,0 +1,127 @@
+<?xml version="1.0"?>
```

Comment:
I would move the config files from `pkg/driver/clickhouse/cluster_config` to `pkg/driver/clickhouse/testdata/cluster_config`

Reply:
Agree.